### ISSUE TITLE
Add external functions.

### ISSUE THREAD

**Issue by schmoelder** [fau-advanced-separations/CADET-Process#8]
In CADET, [Binding Models](https://cadet.github.io/master/modelling/binding/index.html#dependence-on-external-function) and [Reaction Models](https://cadet.github.io/master/modelling/reactions.html#dependence-on-external-function) support the definition of a parameter dependency on external functions. This is currently not implemented in CADET-Process.

**Comment by hannahlanzrath** [comment]
Since we talked about this a few weeks ago - External Functions are currently not implemented in CADET-Process yet, right?

**Comment by schmoelder** [comment]
No, and I don't think I want to do this until we have an improved version in [CADET-Core](https://github.com/cadet/CADET-Core/issues/188).
